uaclient/entitlements/base.py

```diff
@@ -893,7 +905,7 @@ def disable(
                 include other types of reasons in the future.
         """
         msg_ops = self.messaging.get("pre_disable", [])
-        if not util.handle_message_operations(msg_ops):
+        if not util.handle_message_operations(msg_ops, event.info):
```

Comment:
If I got this correctly, event.info will go away here when `disable` implements our progress beauties

Reply:
Yes

Reply:
ack lgtm